src/components/Tag/Tag-story.js

```diff
@@ -27,13 +64,15 @@ storiesOf('Components|Tag', module)
         light background. To use multiple tags use the TagWrapper component.
       `,
     )(() => (
-      <TagWrapper aria-label="Tags">
-        <Tag className="some-class">Offering Managment &middot; 72</Tag>
-        <Tag className="some-class">Development &middot; 63</Tag>
-        <Tag className="some-class">RTP &middot; 51</Tag>
-        <Tag className="some-class">Littleton &middot; 33</Tag>
-        <Tag className="some-class">Dublin &middot; 12</Tag>
-      </TagWrapper>
+      <LightWrapper>
```

Comment:
@stpCollabr8nLstn Should I add the background to `TagWrapper` instead?

Reply:
yeah - that'd be cleaner

Reply:
Will do!